keps/sig-api-machinery/2896-openapi-v3/README.md

```diff
@@ -547,28 +585,40 @@ It's impossible to provide comprehensive guidance, but at the very high level
 These goals will help you determine what you need to measure (SLIs) in the next
 question. -->
 
+This feature should not affect the SLO of any components.
+
+The size of the OpenAPI scales as a cluster has more object types. In OpenAPI V2, aggregating the OpenAPI for all types was very resource intensive. In OpenAPI V3, aggregation is split and the aggregator will act as a proxy instead. We will monitor the current metrics to ensure that no regression is introduced with OpenAPI V3.
+
 ###### What are the SLIs (Service Level Indicators) an operator can use to determine the health of the service?
 
 <!-- Pick one more of these and delete the rest. -->
 
-- [ ] Metrics
-  - Metric name:
-  - [Optional] Aggregation method:
-  - Components exposing the metric:
-- [ ] Other (treat as last resort)
-  - Details:
+A new metric will be added in the aggregator to measure response time for OpenAPI specific requests.
+
+ - [X] Metrics
+  - Metric name: `apiserver_openapi_request_duration_seconds`
+  - Components exposing the metric: kube-apiserver
 
 ###### Are there any missing metrics that would be useful to have to improve observability of this feature?
 
 <!-- Describe the metrics themselves and the reasons why they weren't added
 (e.g., cost, implementation difficulties, etc.). -->
 
+Not at the moment.
+
 ### Dependencies
 
 <!-- This section must be completed when targeting beta to a release. -->
 
 ###### Does this feature depend on any specific services running in the cluster?
 
+OpenAPI V3 aggregates from apiservers provided by APIService.
+
+  - APIService
+    - OpenAPI V3 fetches the `openapi/v3` endpoint and specs from aggregated API
+      - Impact of its outage on the feature: If an APIService is unavailable, the OpenAPI spec of the corresponding APIService will be unavailable but other OpenAPI specs will be unaffected
+      - Impact of its degraded performance or high-error rates on the feature: Same as above
+
```

Comment:
Note: this is better than with openapi/v2 which leads to big cpu load when APIService come and go due to needed spec aggregation.